src/addrman.h

```diff
@@ -671,7 +671,7 @@ class CAddrMan
     std::unordered_map<CNetAddr, int, CNetAddrHash> mapAddr GUARDED_BY(cs);
 
     //! randomly-ordered vector of all nIds
```

Comment:
Perhaps comment _why_ this can be mutable. `vRandom` is unobservable outside the addrman class, so any changes to it are also unobservable (and hence it can be mutated in const methods).

Reply:
Thanks, done